src/app/six-minutes/GameOverModal.tsx

```diff
@@ -0,0 +1,16 @@
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+
+const GameOverModal = () => {
+  return (
+    <div className="absolute w-screen z-10 h-screen backdrop-blur-sm animate-fade">
+      <Card className="absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2">
+        <CardHeader>
+          <h2>🚨 Tiden har gått ut 🚨</h2>
+        </CardHeader>
+        <CardContent>Den gruppen som har tur nå må chugge 🍻</CardContent>
+      </Card>
+    </div>
+  );
+};
```

Comment:
Can we use the `shadcn/ui` component [Alert Dialog](https://ui.shadcn.com/docs/components/alert-dialog) for this instead?

Reply:
I thought about it, but it looked like the only way to open it was with the AlertDialogTrigger? I want it to open automatically after 6 minutes.